code/modules/power/furnace.dm

```diff
@@ -310,3 +295,51 @@ TYPEINFO(/obj/machinery/power/furnace)
 			qdel(W)
 
 		return 1
+
+/datum/action/bar/icon/stuff_mob_into_furnace
+	id = "stuff_into_furnace"
+	duration = 5 SECONDS
+	interrupt_flags = INTERRUPT_ACT | INTERRUPT_STUNNED | INTERRUPT_MOVE | INTERRUPT_ATTACKED
+	var/obj/machinery/power/furnace/furnace = null
+	var/mob/victim = null
+	var/mob/user = null
+
+	New(mob/U, obj/machinery/power/furnace/F, mob/V)
```

Comment:
Please don't use single letter variable names like this, the argument names don't have to be different to the var names in DM, you can just do `src.user = user` and it'll figure it out.